docs/proposals/formatted-output.md

```diff
@@ -0,0 +1,348 @@
+# Formatted ORAS CLI output 
+
+ORAS has prettified output designed for humans. However, for machine processing, especially in some automation scenarios like scripting and CI/CD pipelines, developers want to perform batch operations and chain different commands with ORAS, as well as filtering, modifying, and sorting objects based on the outputs that are emitted by the ORAS command. Developers expect the ORAS output can be generated as machine-readable text not only the prettified or tabular data, so that they can use the formatted outputs like JSON format to perform further advanced operations. 
+
+The formatted output is not intended to supersede the prettified human-readable and friendly output text of ORAS CLI. It aims to provide programming-friendly experience for developers who want to automate their workflows on machines especially on Unix and most Unix-like operating systems, without parsing unstructured text. It will increase the developer experience for ORAS in automation and scripting scenarios. 
+
```

Comment:
Let's add a guidance section to call out the main parameters that would apply to commands. 
Basically most commands could  accept a `--format text|json|yaml|tree|table` but given that we only have json and tree let's only call those out.  

In the guidance section let's use a canonical command say 
`oras manifest fetch --output manifest.json --format json  ghcr.io/oras-project/oras:v1.1.0` as an example

Reply:
Added a Guidance section and used `oras manifest fetch` as an example.